app/forms/embargo_form.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+class EmbargoForm
+  include ActiveModel::Model
+
+  attr_reader :work
+
+  attr_writer :embargoed_until
+  attr_accessor :remove
+
+  validate :embargoed_until_is_valid_date,
+           unless: :remove?
+
+  def initialize(work:, params:)
+    @work = work
+    super(params)
+  end
+
+  def embargoed_until
+    work_embargoed_until = if work.embargoed_until.respond_to?(:strftime)
+                             work.embargoed_until.strftime('%Y-%m-%d')
+                           end
```

Comment:
does this work? `work_embargoed_until = work.embargoed_until.try(:strftime, '%Y-%m-%d')`

Reply:
It works if work.embargoed_until is nil or a date, but it does not work if for some reason it was a string.

Reply:
@rschenk ok, it's fine as-is, I was just curious.